### PR TITLE
Move package, update styles, js, readme

### PR DESCRIPTION
**Pull request by jon-stevens**
This is a component to display the current logged in user, with a dropdown (using the global expander) which displays the email address associated with logged in user, and a log out link. (And in future, some other things).
We are planning on sharing this across a few teams in CA.

Screenshot 2020-10-13 at 18 07 49
Screenshot 2020-10-13 at 18 07 44

### CODE REVIEW

**Comment by jpw** [comment]
Looks very nice and clean!
Have a couple of questions though;
 - SASS `@use` - will this impact the reusability of the component? I know why it's a good thing but i'm wondering how widely it can be adopted
 - I guess we _should_ be using  `@use` everywhere, maybe an issue for open space?
 - why no template file?

**Comment by jon-stevens** [comment]
> Have a couple of questions though;

Good points! 

I have reverted back to imports (for now) but it is probably a good idea to start adopting @use pending team chattage

**Comment by tulipdexter** [comment]
`@use` is not supported in node-sass yet

**Comment by tulipdexter** [comment]
Is it? Wait, are you using dart-sass?

**Comment by tulipdexter** [comment]
> The Sass team discourages the continued use of the @import rule. Sass will gradually phase it out over the next few years, and eventually remove it from the language entirely. Prefer the @use rule instead. (Note that only Dart Sass currently supports @use. Users of other implementations must use the @import rule instead.)

**Comment by jpw** [comment]
Also [dart-sass exists on NPM](https://www.npmjs.com/package/dart-sass) which might be compelling given node-sass can be a pain on occasion. So do we think this is worth discussing in open space?

File: toolkits/springernature/packages/springernature-user-details/view/user-details.hbs [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    <button data-component-user-details-open class="c-user-details__open link-like u-ml10 u-position-right" data-expander data-expander-target="#user">
+        <span data-component-user-first-name>{{first_name}}</span>
+        <span class="u-pr10" data-component-user-last-name>{{last_name}}</span>
+    </button>
+    <div id="user" class="c-user-details__content" data-component-user-details>
+        <span class="c-user-details__content-item u-font-12 u-color-darker-grey">Signed in as</span>
+        <div class="c-user-details__content-item c-user-details__email u-mb10 u-font-12" data-component-user-email>{{email_address}}</div>
+        <ul class="c-user-details__links">
+            <li class="c-user-details__link u-cursor-pointer">
+                <a class="c-user__logout u-display-block u-font-12" href="{{logoutUri}}"data-component-logout>Log out</a>
```
Comment (jpw) [inline toolkits/springernature/packages/springernature-user-details/view/user-details.hbs:11]:
```suggestion
                <a class="c-user__logout u-display-block u-font-12" href="{{logoutUri}}" data-component-logout>Log out</a>
```

**Review by jpw — APPROVED** [review]
nice!

**Pull request merged**